recipes/gnuplot/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set pkgname = "gnuplot" %}
+{% set version = "5.0.4" %}
+{% set sha256 = "151cb845728bde75eb9d1561b35140114a05a7c52a52bd35b4b2b3d944e0c31e" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/{{ pkgname }}/files/{{ pkgname }}/{{ version }}/{{ pkgname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - llvm  # [osx]
+    - pkg-config  # [osx]
+    - libgd
+    - ncurses
+    - cairo
+    - readline
+    - qt
+  run:
+    - libgcc  # [not osx]
```

Comment:
@conda-forge/core What exactly is the procedure with respect to `libgcc` etc? I can't keep track 😆